### PR TITLE
Fix gas price issue

### PR DESCRIPTION
**Pull request by jonathansmirnoff**
This PR fixes issue #7. Also it sets a fixed version of web3 to avoid issues when web3 npm package is updated.

### CODE REVIEW

**Comment by jonathansmirnoff** [comment]
@alebanzas you're right. Let me figure out how to do it.

**Comment by alebanzas** [comment]
> @alebanzas you're right. Let me figure out how to do it.

@ilanolkies can help with the query.

**Comment by ilanolkies** [comment]
Hi!

Here's how rsk faucet gets gas price: https://github.com/rsksmart/faucet/pull/12
More info about min gas price: https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP09.md

**Comment by jonathansmirnoff** [comment]
I implemented the logic to get the GP. Please check it, thanks!

File: src/operations.js [hunk 1]
```diff
@@ -39,30 +39,32 @@ export const dispense = () => dispatch => {
...
-        'outputs': [],
-        'payable': false,
-        'stateMutability': 'nonpayable',
-        'type': 'function',
-        'signature': '0x5f746233'
-      }
-    ], config.faucet);
+  getMinimunGasPrice().then( gmp => {
+    window.ethereum.enable()
+    .then(accounts => {
```
Comment (ilanolkies) [inline src/operations.js]: We should chain promises. I change it.

File: src/operations.js [hunk 2]
```diff
@@ -80,4 +82,14 @@ export const getNetwork = () => dispatch => {
 
     dispatch(receiveNetwork(res));
   });
-};
\ No newline at end of file
+};
+
+async function getMinimunGasPrice(){
+  const web3 = new Web3(config.publicNode);
+  var block = await web3.eth.getBlock("latest");
```
Comment (ilanolkies) [inline src/operations.js]: Chaining here too.

**Pull request merged**